ezpublish/config/routing.yml

```diff
@@ -1,3 +1,7 @@
+acme_ez_platform_default_site:
+    resource: "@AcmeEzPlatformDefaultSiteBundle/Resources/config/routing.yml"
+    prefix:   /
+
```

Comment:
as said in gtalk i would let the users to decide if they want install acme bundle or not.